extensions/collection-assets/README.md

```diff
@@ -0,0 +1,38 @@
+# Collection Assets Extension Specification
+
+- **Title: Collection Assets**
+- **Identifier: collection-assets**
+- **Field Name Prefix: -**
+- **Scope: Collection**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+A Collection extension to provide a way to specify assets available on the collection-level.
+
+- [Example](examples/example-esm.json)
+- [JSON Schema](json-schema/schema.json)
+
+This extension introduces a single new field, `assets` at the top level of a collection.
+An Asset Object defined at the Collection level is the same as the [Asset Object in Items](../../item-spec/item-spec.md#asset-object).
+
+Collection-level assets MUST NOT list any files also available in items.
```

Comment:
Why not? This sounds hard to enforce and like it won't hurt much if someone lists an asset in two places

Reply:
We want to avoid people put all their Item assets into collections. Indeed that's hard to enforce.

Reply:
I have a preference against *MUST*s that can't be expressed in the JSON schema and that require a whole catalog of context to validate. I think this should be demoted to a best practice.

Reply:
Well, we have quite a lot of things in the spec that can't be validated with JSON Schema though.

Tagging @cholmes as he was the main driver for this... I personally don't care so much about this rule.

Reply:
I'm fine with a 'should' / best practice here.

Reply:
```suggestion
Collection-level assets SHOULD NOT list any files also available in items.
```

Reply:
Done.